agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4534,14 +4537,41 @@ bool slave_thread::handle_ap_metrics_query(Socket *sd, ieee1905_1::CmduMessageRx
         auto bssid = ap_metrics_query_tlv->bssid_list(bssid_idx);
         if (!std::get<0>(bssid)) {
             LOG(ERROR) << "Failed to get bssid " << bssid_idx << " from AP_METRICS_QUERY";
-            return false;
+            //return false;
         }
         LOG(DEBUG) << "Received AP_METRICS_QUERY_MESSAGE, mid=" << std::hex << int(mid)
                    << "  bssid " << std::get<1>(bssid);
     }
+    LOG(DEBUG) << "Received AP_METRICS_QUERY_MESSAGE, mid=" << std::hex
+               << int(mid); //TODO: remove this string after solving problem with query tlv.
+    auto cmdu_header = cmdu_tx.create(mid, ieee1905_1::eMessageType::AP_METRICS_QUERY_MESSAGE);
+
+    if (!cmdu_header) {
+        LOG(ERROR) << "Failed building IEEE1905 AP_METRICS_QUERY_MESSAGE";
+        return false;
+    }
+    message_com::send_cmdu(monitor_socket, cmdu_tx);
```

Comment:
I've also used `message_com::forward_cmdu_to_uds` here to forward message to monitor thread.